src/components/select/_select.scss

```diff
@@ -146,7 +148,11 @@
       box-shadow: none;
 
       &:hover {
-        background-color: $color__white;
+        @if feature-flag-enabled('components-x') {
```

Comment:
The code here seems in stable mix-in, thus `feature-flag-enabled('components-x')` never is `true` here?

Similar question at:
* [`.bx--tag--ibm`](https://github.com/IBM/carbon-components/pull/1643/files#diff-b7d97cbc5db381fb59b49753b4732a44R29)

Reply:
Good catch, thanks @asudoh!